homeassistant/components/flux_led/config_flow.py

```diff
@@ -153,18 +160,20 @@ async def async_step_prompt_options(self, user_input=None):
                     CONF_HOST: user_input[CONF_HOST],
                     CONF_NAME: device_name,
                 }
-
                 config_data = self._config_entry.data.copy()
                 config_data[CONF_DEVICES][device_id] = device_data
 
+                self.hass.config_entries.async_update_entry(self._config_entry, data={})
+
                 self.hass.config_entries.async_update_entry(
                     self._config_entry, data=config_data
                 )
 
+                async_dispatcher_send(
+                    self.hass, SIGNAL_ADD_DEVICE, {device_id: device_data}
+                )
                 options_data = self._config_entry.options.copy()
                 options_data["global"] = self._global_options
```

Comment:
Changing the variable here doesn't have any effect since it's overwritten below.

Reply:
Removed.